crates/aptos/src/move_tool/mod.rs

```diff
@@ -542,8 +535,13 @@ impl CliCommand<&'static str> for TestPackage {
             compiler_config: CompilerConfig {
                 known_attributes: known_attributes.clone(),
                 skip_attribute_checks: self.move_options.skip_attribute_checks,
+                bytecode_version: fix_bytecode_version(
+                    self.move_options.bytecode_version,
+                    self.move_options.language_version,
+                ),
                 compiler_version: self.move_options.compiler_version,
                 language_version: self.move_options.language_version,
+                experiments: experiments_from_opt_level(&self.move_options.optimization_level),
```

Comment:
This is experimental, can we mark it as such?

Reply:
The "experiment" feature should really is just a different (untyped, boolean) form of configuration.  It has been convenient to select experiments, but to reduce the amount of plumbing (at Wolfgang's request) I'm trying to use it to control other features, such as which passes get run for different optimization levels.

Reply:
It is not 'experimental', it is an 'experiment' which is in some processes just a feature flag. It is the way how the compiler is controlled.